projectfiles/Xcode/README.md

```diff
@@ -30,33 +31,66 @@ Builds for maximum (runtime) speed and compatibility; it builds for 64-bit, with
 * **Debug**:
 Builds for maximum compiling speed, and uses the current OS as the SDK. If you just want to compile for testing things yourself, this is the way to go.
 
-## Step by step building
-1. Obtain sources from github using `git clone https://github.com/wesnoth/wesnoth` or use your favourite git program
-2. Use `Finder` to navigate into `wesnoth/projectfiles/Xcode` . And now, there will be 2 options.
-3. Double click on `Fix_Xcode_Dependencies` script and wait until it will be done.
-5. (Optional) Compile translations as it is described in **Translations** section.
-6. Now you can open `The Battle for Wesnoth.xcodeproj` file in Xcode.
-
 ## Translations
-To compile translations you need `gettext-tools` and `scons`. You can obtain these tools using `brew install gettext scons`. In the Terminal, `cd` to the Wesnoth root directory, and run `scons translations`. This will compile all the translations into a translations directory.
+To compile translations you need `gettext-tools` and `scons`. You can obtain these tools using `brew install gettext scons`. You also have to force-link Homebrew's `gettext` tools using `brew link --force gettext` command. In the Terminal, `cd /PATH/TO/PROJECT` and run `scons translations`. This will compile all the translations into a translations directory.
 
 ## Packaging
-When compiling Wesnoth for an official release, the following steps should be taken:
+When compiling Wesnoth for an official release, the following steps should be taken. Packaging is separated to the 4 subchapters:
+1. In first chapter we will prepare package for all 3 distribution sources.
+2. In second chapter we will notarize Steam package and we will release it to the SteamStore.
+3. In third chapter we will notarize SourceForge package, we will create `.dmg` image and we will release it to the SourceForge.
+4. In fourth chapter we will release package to the Mac AppStore.
 
+### Packaging - Packages preparation
+ * Obtain sources from github using `git clone -b BRANCH --depth 10 https://github.com/wesnoth/wesnoth /PATH/TO/PROJECT` or use your favourite git program.
+ * Checkout to the latest tag using `cd /PATH/TO/PROJECT ; git checkout TAG` or use your favourite git program.
+ * Use `Finder` to navigate into `/PATH/TO/PROJECT/projectfiles/Xcode`.
+ * Double click on `Fix_Xcode_Dependencies` script and wait until it will be done.
```

Comment:
I'd say it as "Double click on **the** `Fix_Xcode_Dependencies` script and wait until it **finishes**."